decayAmplitude/ampIntegralMatrix.cc

```diff
@@ -31,7 +31,15 @@
 //-------------------------------------------------------------------------
 
 
+#include <boost/version.hpp>
+#if (BOOST_VERSION >= 107200)
+#include <boost/timer/progress_display.hpp>
+using boost::timer::progress_display;
+#else
 #include <boost/progress.hpp>
+using boost::progress_display;
+#endif
+
```

Comment:
Could we move this code to one header file, as it appears multiple times?

Reply:
Yes, it appears really often ;)

Reply:
Yes, I agree it is ugly now.